German translation

dimag0g opened the pull request:
Here's a German translation I wish to contribute, and I'd like to ask everyone to invite German-speaking reviewers they might know. My German is pretty basic, so I just machine-translated most of the phrases, trying to fix them the best I could, especially regarding game terminology and the use of pronouns.

As with French, I tried to stick to GoG version terminology for:
- world map locations
- artifacts
- monsters
- spells
- resources

German has the same "polite you" feature as French, so:
- the game addresses the player with polite "you"
- the player's subjects in the game use polite "you"
- the enemies and unfriendly event characters use regular "you"

Now, I suppose my work is filled with all kinds of mistakes. I'd like to ask reviewers to concentrate on recurring issues in the first place, such as:
- wrong verb tense being used in certain phrase types
- wrong choice of recurring words, such as creature/monster/unit, view/show, allow/enable, etc

I'm not trying to sneak poor work in, I just want to say that fixing every single typo in this PR is going to take a very long time, so it would be more efficient to accept a translation with a bunch of mistakes, and then make additional PRs fixing them: that would allow several people to work in parallel, without the need to merge 3000+ lines.

a1exsh commented:
@dimag0g I will have a look :+1:

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "High morale enables the %{monster} to attack again."
-msgstr ""
+msgstr "Eine hohe Moral ermöglicht den %{monster} erneut anzugreifen."
```
```suggestion
msgstr "Hoher Kampfgeist ermöglicht es, dass %{monster} erneut angreifen."
```
Alternatively: "erlaubt dem/der ... anzugreifen".

a1exsh reviewed (commented):
10% through

a1exsh commented:
Now this PR is too big to work with in the browser.  Tried Github Desktop, but I don't see any way to review pull requests — did I miss it, or it's not there yet?..

oleg-derevenetz commented:
> Tried Github Desktop, but I don't see any way to review pull requests — did I miss it, or it's not there yet?..

Github Desktop still [has no ability](https://github.com/desktop/desktop/issues/11517) to review PRs.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "The mirror image is created."
-msgstr ""
+msgstr "Das Spiegelbild ist zerstört!"
```
```suggestion
msgstr "Das Spiegelbild ist herbeigerufen."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Please select another hero."
-msgstr ""
+msgstr "Bitte wähle einen anderen Helden aus."
```
```suggestion
msgstr "Bitte wählen Sie einen anderen Helden aus."
```
The polite form.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Left Turret"
-msgstr ""
+msgstr "Turm Links"
```
```suggestion
msgstr "Linker Turm"
```
Linker/Rechter

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "The %{name} fires with the strength of %{count} Archers"
-msgstr ""
+msgstr "Der %{name} feuert mit der Kraft von %{count} Bogenschützen"
```
Ah, here it is tricky in German.  If `%{name}` is to be replaced with either of "Left Turret" or "Right Turret", we should move the article (Der) to that translation, i.e: `%{name} feuert mit ...`, where `%{name}` is `Der linke Turm`.

dimag0g replied:
I think this sentence is only ever used with turrets, so the article is always "Der".

a1exsh replied:
Then we should adapt the "Left/Right Turret" translations accordingly to become "linke/rechte Turm".

zenseii replied:
I cannot help with the German grammar, but for Norwegian I ended up writing what translates to "The castle's %{name} fires with the...", because changing Left/Right Turret to definite form in their definite forms was not an option since this string is also used in the build screen/castle options and the space there is limited, not to mention that none of the other buildings had the definite article in their names.

Another point for not including the definite article in the turret string was because the adjectives right/left would change whether they were definite or indefinite in Norwegian.

dimag0g replied:
@a1exsh Yep, "Right Tower" is aslo used as a dialog title in the building interface. I'd say an extra capital letter is the least evil we can get.

Which reminds me: some of the strings (mostly monster names) are reused by being converted to lower case in the game, so that you see "Elves" in the upgrade dialog but "The elves did 125 damage" during a battle. This is already a problem in Russian, as Russian letters have no capitals in "C" locale, and it will indeed ruin German capitalization as well.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "%{count} %{name} rise(s) from the dead!"
-msgstr ""
+msgstr "%{count} %{name} auferstehen von den Toten!"
```
```suggestion
msgstr "%{count} %{name} auferstehen aus den Toten!"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Elven Alliance"
-msgstr ""
+msgstr "Elfen-Buendnis"
```
```suggestion
msgstr "Elfen-Bündnis"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Kraeger defeated"
-msgstr ""
+msgstr "Kraeger besiegt"
```
```suggestion
msgstr "Kräger besiegt"
```
Likely, the original meant to be with umlaut, but it's not common in English.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "will spend most of their time fighting with one another.  Victory is yours "
 "when you have defeated all of their castles and heroes."
 msgstr ""
+"Roland braucht dich, um die Herren in der Nähe seiner Burg zu besiegen, um "
```
So what about the polite "you" here?

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "out fighting. You will need to own all four castles in this small valley to "
 "win."
 msgstr ""
+"Ihre Feinde sind gegen Ihnen verbündet und fangen in der Nähe an, also sei "
```
```suggestion
"Ihre Feinde sind gegen Sie verbündet und fangen in der Nähe an, also seien Sie "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "This is the final battle. Both you and your enemy are armed to the teeth, "
 "and all are allied against you. Capture Archibald to end the war!"
 msgstr ""
+"Dies ist die letzte Schlacht. Sowohl du als auch dein Feind sind bis an die "
```
Can't help to notice that English is much easier to translate 1:1 to German, than to Russian.  Machine translation is pretty good here.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Necromancers"
-msgstr ""
+msgstr "Totenbeschwoerer"
```
```suggestion
msgstr "Totenbeschwörer"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Slay the Dwarves"
-msgstr ""
+msgstr "Schlagt die Zwerge"
```
```suggestion
msgstr "Schlachtet die Zwerge"
```
Slay, not slap. ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Country Lords"
-msgstr ""
+msgstr "Landesfuersten"
```
```suggestion
msgstr "Landesfürsten"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Dragon Master"
-msgstr ""
+msgstr "Drachen-Meister"
```
```suggestion
msgstr "Drachenmeister"
```
There are other places where use of `-` is not German native.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Greater Glory"
-msgstr ""
+msgstr "Groesste Ehre"
```
```suggestion
msgstr "Größte Ehre"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "enemy castles and there are no more heroes left to fight."
 msgstr ""
+"König Archibald verlangt von Ihnen, dass Sie die drei Feinde in dieser "
+"Region besiegst.  Sie sind nicht miteinander verbündet, also werden sie den "
```
```suggestion
"Region besiegen.  Sie sind nicht miteinander verbündet, also werden sie den "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
+"Sie müssen die Barbarenstämme des Nordens vereinen, indem Sie sie erobern. "
+"Wie in der vorherigen Mission ist der Feind nicht mit Ihnen verbündet, aber "
+"er hat mehr Ressourcen zur Verfügung. Sie warden gewinnen, wenn Sie alle "
+"feindlichen Burgen besitzen und keine Helden mehr zum Kämpfen übrig sind."
```
```suggestion
"Wie in der vorherigen Mission sind die Feinde nicht miteinander verbündet, aber "
"sie haben mehr Ressourcen zur Verfügung. Sie werden gewinnen, wenn Sie alle "
"feindlichen Burgen besitzen und keine Helden mehr zum Kämpfen übrig sind."
```

dimag0g replied:
Would "die haben" also be appropriate instead of "sie haben"?

a1exsh replied:
> Would "die haben" also be appropriate instead of "sie haben"?

Maybe, but I'm not certain about that.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "%{name} the %{race}"
-msgstr ""
+msgstr "%{name} der %{race}"
```
Getting genders right will not be possible given existing translation strings. The source code will have to be reworked.

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "have no castle and must take one within 7 days, or lose this battle. (Hint: "
 "The nearest castle is to the southeast.)"
 msgstr ""
+"Gutmenschen-Zauberer haben die Burg der Nekromanten eingenommen. Ihr müsst "
```
"Sie müssen ..."

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "\"I will accept your surrender and grant you and your troops safe passage "
 "for the price of %{price} gold.\""
 msgstr ""
+"\"Ich werde Ihre Kapitulation akzeptieren und Ihnen und Ihren Truppen eine "
```
Sure, I intended to use a non-polite form here.

a1exsh reviewed (changes requested):
I notice a lot of polite/non-polite "you" mismatch and missing umlauts: these should be easy to spot even for a beginner.

Another pass on proof-reading internally would be helpful before opening a PR, IMO. ;)

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Morale"
-msgstr ""
+msgstr "Moral"
```
I like "Kampfgeist", but I'm not sure if it will fit in the interface everywhere instead of "Moral". Furthermore, the GoG version uses "Moral" as well:
![Untitled](https://user-images.githubusercontent.com/10945319/148639584-bacd7889-d475-4450-b622-3eca429eef34.png)

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "Surrendering costs gold. However if you pay the ransom, the hero and all of "
 "his or her surviving creatures will be available to recruit again."
 msgstr ""
+"Das Aufgeben kostet Gold. Wenn Sie jedoch das Lösegeld bezahlen, können der "
+"Held und alle seine überlebenden Kreaturen wieder rekrutiert werden."
```
What's wrong with the existing translation?

a1exsh replied:
I'm trying to make it clear that you don't need to recruit the suviving monsters separately, which is the impression that I have after reading your variant.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "The Sphere of Negation artifact is in effect for this battle, disabling all "
 "combat spells."
 msgstr ""
+"Das Artefakt \"Sphäre der Verneinung\" ist in diesem Kampf wirksam und macht "
+"alle Kampfzauber unwirksam."
```
"ist in diesem Kampf aktiv"?

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Wait this unit"
-msgstr ""
+msgstr "Auf dieses Zug warten"
```
Shall I be using "Runde" everywhere instead of "Zug"?

a1exsh replied:
I thought of that initially, but there seems to be a difference between the two.  A Round (Runde) comprises moves in turn by each of the creatures (Zug).  Let's try to stick to this logic.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "Waits the current creature. The current creature delays its turn until after "
 "all other creatures have had their turn."
 msgstr ""
+"Wartet die aktuelle Kreatur. Die aktuelle Kreatur verzögert ihren Zug, bis "
```
"Die aktuelle Kreatur wartet und verzögert ihren Zug, ..."?

a1exsh replied:
Also possible.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Skip this unit"
-msgstr ""
+msgstr "Dieses Zug überspringen"
```
I thought "aussetzen" is "defer", meaning it's closer to "wait", while "übergehen/überspringen" is closer to "skip".

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "The %{spell} does %{damage} damage to all living creatures."
 msgstr ""
+"Der %{spell} verursacht %{damage} Schaden bei allen lebenden Kreaturen."
```
I'll use "an" when appropriate.

dimag0g commented:
@a1exsh Let me know if you would like to continue the review like this. Alternatively, I could commit your fixes I've done so far, we get this merged, and then you submit the rest of your fixes as a separate PR. I suppose it will be easier to review this second PR with the fixes.

a1exsh commented:
@dimag0g so far I've managed to read about 25% of the change.  A number of issues are repeating, so I was trying not to raise the same issue many times, but it would help if you could check for other instances of the same and fix them pro-actively.

I would try to continue reading further from where I left, github permitting ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 
 msgid "Please Remember"
-msgstr ""
+msgstr "Bitte erinnern"
```
```suggestion
msgstr "Bitte merken"
```

dimag0g commented:
@a1exsh Ok, I already pushed fixes I did so far, some of them pro-active. I will wait for more comments from reviewers.
Thanks a lot for fixing my mess.

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 msgstr ""
+" Taste auf der Tastatur.\n"
+"\n"
+"Viel Spaß mit dem Spiel!"
```
```suggestion
"Viel Spaß beim Spielen!"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 
 msgid "Find the ultimate artifact."
-msgstr ""
+msgstr "Das ultimate Artefakt finden."
```
```suggestion
msgstr "Das ultimative Artefakt finden."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 
 msgid "Lose the hero: %{name}."
-msgstr ""
+msgstr "Den Helden: %{name} verlieren."
```
```suggestion
msgstr "Den Helden verlieren: %{name}."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 msgid "Fail to win by the end of month %{month}, week %{week}, day %{day}."
 msgstr ""
+"Gewinnen Sie nicht bis zum Ende des Monats %{month}, der Woche %{week}, des "
+"Tages %{day}."
```
```suggestion
"Nicht bis zum Ende des Monats %{month}, der Woche %{week}, des Tages %{day} gewinnen."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 
 msgid "Click to return to the main menu."
-msgstr ""
+msgstr "Klicken Sie auf , um zum Hauptmenü zurückzukehren."
```
```suggestion
msgstr "Klicken, um zum Hauptmenü zurückzukehren."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -2789,307 +3136,372 @@ msgid ""
 
 msgid "Astrologers proclaim Month of the %{name}."
-msgstr ""
+msgstr "Astrologen rufen den Monat des %{name} aus."
```
```suggestion
msgstr "Astrologen verkündigen den Monat des %{name}."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 
 msgid "%{monster} population increases by +%{count}."
-msgstr ""
+msgstr "Die Bevölkerung von %{monster} Bevölkerung steigt um +%{Anzahl}."
```
```suggestion
msgstr "Die Bevölkerung von %{monster} steigt um +%{Anzahl}."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 "banished from this land."
 msgstr ""
+"%{color}er Spieler, dies ist Ihren letzter Tag, um eine Stadt zu erobern, "
+"oder werden Sie aus diesem Land verbannt."
```
```suggestion
"%{color}er Spieler, dies ist Ihr letzter Tag, um eine Stadt zu erobern, "
"oder Sie werden aus diesem Land verbannt."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 "%{color} player, your heroes abandon you, and you are banished from this "
 "land."
 msgstr ""
+"%{color}er Spieler, Ihre Helden lassen Ihnen im Stich, und Sie werden aus "
```
```suggestion
"%{color}er Spieler, Ihre Helden lassen Sie im Stich, und Sie werden aus "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 
 msgid "Bring up the file options menu, alloving you to load menu, save etc."
 msgstr ""
+"Ruft das Menü \"Dateioptionen\" auf, in dem Sie das Menü laden, speichern "
```
```suggestion
"Ruft das Menü \"Dateioptionen\" auf, in dem Sie das Spiel laden, speichern "
```

a1exsh replied:
The original text also needs correction.

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 
 msgid "Are you sure you want to overwrite the save with this name?"
 msgstr ""
+"Sind Sie sicher, dass Sie den Speicher mit diesem Namen überschreiben wollen?"
```
```suggestion
"Sind Sie sicher, dass Sie die Datei mit diesem Namen überschreiben wollen?"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 
 msgid "Try looking on land!!!"
-msgstr ""
+msgstr "Versuchen Sie, an Land zu suchen!!!"
```
```suggestion
msgstr "Versuchen Sie es, auf dem Land zu suchen!!!"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -3098,131 +3510,161 @@ msgid_plural ""
 
 msgid "Nothing here. Where could it be?"
-msgstr ""
+msgstr "Hier ist nichts. Wo könnte es sein?"
```
```suggestion
msgstr "Hier ist nichts. Wo könnte es nur sein?"
```
A bit more liveliness ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 
 msgid "Natasha"
-msgstr ""
+msgstr "Natasha"
```
```suggestion
msgstr "Natascha"
```
Vielliecht

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 
 msgid "You cannot pick up this artifact, you already have a full load!"
 msgstr ""
+"Sie können dieses Artefakt nicht aufheben, Sie haben bereits eine volle "
```
```suggestion
"Sie können dieses Artefakt nicht aufsammlen, Sie haben bereits eine volle "
```
Otherwise it could mean that it's too heavy :D

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 msgid "Ransacking an enemy camp, you discover a hidden cache of treasures."
 msgstr ""
+"Bei der Plünderung eines feindlichen Lagers entdecken Sie ein verstecktes "
+"Versteck mit Schätzen."
```
"verstecktes Versteck"...

dimag0g replied:
I'll use "verborgen"

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "come back next week for more.\""
 msgstr ""
+"Der Wächter der Mühle verkündet:\n"
+"\"Mylord, ich habe sehr hart gearbeitet, um Ihnen diese Ressourcen zur "
```
```suggestion
"\"Mein Herr, ich habe sehr hart gearbeitet, um Ihnen diese Ressourcen zur "
```
?

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
+"Sie fangen einen Kobold, der dummerweise inmitten eines Haufens von "
+"Zauberpilzen schläft.\n"
+"Im Austausch für seine Freiheit, führt er Sie zu einem kleinen Topf mit "
+"wertvollen Dingen gefüllt."
```
```suggestion
"Im Austausch für seine Freiheit, führt er Sie zu einem kleinen Topf "
"voller wertvollen Dinge."
```

a1exsh replied:
Hm, at the same time "voller wert-_vollen_"...

dimag0g replied:
"mit wertvollen Dingen drin"?

a1exsh commented:
It would be nice if some native German speaker could have a look as well and suggest corrections for non-idiomatic stuff.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "The dragons, snarling and growling, agree to join forces with you, their "
 "'Ally'."
 msgstr ""
+"Die Drachen, knurrend und knurrend, stimmen zu, sich mit Sie, ihrem "
```
"knurrend und grollend"?

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "'Ally'."
 msgstr ""
+"Die Drachen, knurrend und knurrend, stimmen zu, sich mit Sie, ihrem "
+"\"Verbündeten\", zusammenzuschließen."
```
I suggest we do this as a separate PR. This change should apply to French (`« bhah »`) and Russian (`«blah»`) as well, and maybe to other languages.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Defender Helm"
-msgstr ""
+msgstr "Verteidiger Helm"
```
I think it was supposed to be "Verteidigungshelm des Schutzes" or just "Verteidigungshelm", I'll have to check in the game.

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "safe terrain."
 msgstr ""
+"Sie stoßen auf einen alten Wagen, der von einem Händler zurückgelassen "
+"wurde, der es nicht ganz auf sicheres Terrain geschafft hat."
```
```suggestion
"wurde, der es nicht ganz auf sicheres Land geschafft hat."
```

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Minor Scroll"
-msgstr ""
+msgstr "Kleine Schriftrolle"
```
That's how it is in the GoG version.

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 
 msgid "As you drink the sweet water, you gain luck for your next battle."
 msgstr ""
+"Wenn Sie das süße Wasser trinken, erhalten Sie Glück für Ihren nächsten "
```
```suggestion
"Als Sie das süße Wasser trinken, erhalten Sie Glück für Ihren nächsten "
```

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Thunder Mace"
-msgstr ""
+msgstr "Donnerstab"
```
![Untitled2](https://user-images.githubusercontent.com/10945319/148651031-7b8be76c-b494-4851-8d69-2c5fc4c06830.png)

a1exsh replied:
OK, then let's keep it as "Donnerstab".

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Cannot build. Already built here this turn."
-msgstr ""
+msgstr "Kann nicht bauen. Sie haben hier in dieser Runde bereits etwas gebaut."
```
I'm collecting suggestions to improve English text as well. I'll do a separate PR once I have finished this one. Thanks for the suggestion.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Farm"
-msgstr ""
+msgstr "Farm"
```
Sure, but:
![Untitled](https://user-images.githubusercontent.com/10945319/148651260-4c1f6a9d-009a-4f51-8e71-90cfd781899e.png)

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Armory"
-msgstr ""
+msgstr "Waffenfabrik"
```
The screenshot above has Waffenfabrik as well.

dimag0g commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Sprites"
-msgstr ""
+msgstr "Kobolde"
```
I actually hate "Kobolde", for me they are closer to goblins than sprites/fairies. Perhaps let's change just this one.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Pen"
-msgstr ""
+msgstr "Verschlag der Eber"
```
GoG version terminology

a1exsh replied:
No one is perfect :joy:

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "there and becomes more vulnerable to attack."
 msgstr ""
+"Der Burggraben verlangsamt angreifende Einheiten. Jede Einheit, die den "
+"Graben betritt, muss ihren Zug dort beenden und wird anfälliger für Angriffe."
```
I wonder if I should use Runde or Zug.

a1exsh replied:
"Zug" seems OK here.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Not enough resources to buy monsters."
-msgstr ""
+msgstr "Nicht genug Ressourcen für den Kauf von Monstern."
```
I'll take a note about consistency in English as well.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "Do you accept?"
 msgstr ""
+"Das %{monster} lässt sich von Ihrer diplomatischen Zunge überzeugen und "
+"bietet Ihnen an, sich Ihre Armee für die Summe von %{gold} Gold "
```
Nope, it's singular:

```
    if ( troop.GetCount() == 1 ) {
        message = _( "The %{monster} is swayed by your diplomatic tongue, and offers to join your army for the sum of %{gold} gold.\nDo you accept?" );
    }
    else {
        message = _( "The creatures are swayed by your diplomatic\ntongue, and make you an offer:\n \n" );
```

a1exsh replied:
Ah, OK, I didn't think there is a special case for just 1 monster.

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "gold.\n"
 "Do you accept?"
 msgstr ""
+"Alle %{offer} des %{monster} werden sich Ihre Armee für die Summe von "
```
It's up to the game to use the plural form here, so it's ok.

dimag0g replied:
> It would be nice if some native German speaker could have a look as well and suggest corrections for non-idiomatic stuff.

They'll come complaining once the translation is released :)

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 "Difficulty"
 msgstr ""
+"Karte\n"
+"Schwierigkeit"
```
What about "Spiel\nSchwierigkeit"?

a1exsh replied:
Sure, please check all similar places.  I'm trying not to correct every place I find, if I already raised it elsewhere.

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 "to buy the maps?"
 msgstr ""
+"Ein Hauptmann im Ruhestand, der auf dieser renovierten Fischerplattform "
+"lebt, bietet Ihnen an, Ihnen für 1.000 Gold Karten des Meeres zu verkaufen, "
```
```suggestion
"lebt, bietet Ihnen an, für 1.000 Gold die Karten des Meeres zu verkaufen, "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 "to give my maps away for free!\""
 msgstr ""
+"Der Hauptmann seufzt. \"Du hast nicht genug Geld, was?  Du kannst nicht "
+"erwarten, dass ich meine Karten umsonst weggebe!\""
```
```suggestion
"erwarten, dass ich meine Karten umsonst ausgebe!\""
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 
 msgid "it whispers. (sniff) \"Well, come back when you can pay me.\""
 msgstr ""
+"flüstert es. (schnieft) \"Na, dann komm wieder, wenn du mich bezahlen kannst."
```
```suggestion
"flüstert er. (schnieft) \"Na, dann komm wieder, wenn du mich bezahlen kannst."
```
Der Baum.

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 
 msgid "Upon your approach, the tree opens its eyes in delight."
-msgstr ""
+msgstr "Wenn Sie sich nähern, öffnet der Baum erfreut seine Augen."
```
```suggestion
msgstr "Als Sie sich nähern, öffnet der Baum erfreut seine Augen."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
+"\""
 
 msgid ""
 "Nestled among the trees sits a blind seer. After explaining the intent of "
```
Here the original text also could benefit from a correction.  I believe it is the hero who explains the intent of his journey, not the seer. ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 "strengths and weaknesses of your opponents."
 msgstr ""
+"Eingebettet zwischen den Bäumen sitzt ein blinder Seher. Nachdem er Ihnen "
+"das Ziel Ihrer Reise erklärt hat, aktiviert der Seher seine Kristallkugel, "
```
```suggestion
"Eingebettet zwischen den Bäumen sitzt ein blinder Seher. Nachdem Sei ihm "
"das Ziel Ihrer Reise erklärt haben, aktiviert der Seher seine Kristallkugel, "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5003,197 @@ msgid ""
 "gold."
 msgstr ""
+"Nachdem Sie die Diener des Dämons besiegt haben, finden Sie einen "
+"versteckten Cache mit %{count} Gold."
```
Need to translate "Cache".

dimag0g replied:
"verborgenen Versteck" again :)

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 "That spell costs %{mana} mana. You only have %{point} mana, so you can't "
 "cast the spell."
 msgstr ""
+"Dieser Zauberspruch kostet %{Mana} Mana. Sie haben nur %{Punkt} Mana, also "
```
```suggestion
"Dieser Zauberspruch kostet %{mana} Mana. Sie haben nur %{point} Mana, also "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "You cannot dismiss a hero in a castle"
-msgstr ""
+msgstr "Man kann einen Helden nicht in eine Burg entlassen"
```
```suggestion
msgstr "Man kann einen Helden nicht in einer Burg entlassen"
```

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Att."
-msgstr ""
+msgstr "Angriff"
```
Honestly I didn't check. Let's go with "Angr." / "Vert." / "Kraft" / "Wissen".

a1exsh replied:
> Honestly I didn't check. Let's go with "Angr." / "Vert." / "Kraft" / "Wissen".

Sure.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Your hero is too tired to cast this spell today. Try again tomorrow."
 msgstr ""
+"Ihren Held ist zu müde, um diesen Zauber heute zu wirken. Versuchen Sie es "
```
```suggestion
"Ihr Held ist zu müde, um diesen Zauber heute zu wirken. Versuchen Sie es "
```

dimag0g commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3133 @@
 
 msgid "Gems, Cr, Slf & Mer:"
-msgstr ""
+msgstr "Est, Kr, Sch & Que:"
```
I was out of ideas so I overtook the GoG translation here:
![Untitled](https://user-images.githubusercontent.com/10945319/148653777-cd766016-9eab-4ed3-9b77-b2b9bc87dea6.png)

We could go with "Edl" or "Juw" if you prefer.

a1exsh replied:
We can also just say "Andere Rohstoffe" instead of "Est, Kr, Sch & Que".

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 "work."
 msgstr ""
+"Sie müssen sich innerhalb von %{count} Feldern eines Monsters befinden, "
+"damit der Vision wirkt."
```
```suggestion
"damit die Vision wirkt."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 msgid ""
 "Your hero produces %{count} gold pieces per turn as tax revenue from estates."
 msgstr ""
+"Ihre Held produziert %{count} Goldstücke pro Runde als Steuereinnahmen aus "
```
```suggestion
"Ihr Held produziert %{count} Goldstücke pro Runde als Steuereinnahmen aus "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Orange"
-msgstr ""
+msgstr "Orange"
```
This color will not work with automated suffix `%{color}e(r|s)`...

dimag0g replied:
We may need another context, i.e. "player|Orange" and drop the suffixes for good. I would postpone it for a different PR though.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Purple"
-msgstr ""
+msgstr "Purpur"
```
```suggestion
msgstr "Lila"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Ore Mine"
-msgstr ""
+msgstr "Erz Mine"
```
```suggestion
msgstr "Erzmine"
```

dimag0g replied:
Does this apply to other resources as well, e.g. Kristallmine?

dimag0g replied:
Does this apply to other resources as well, e.g. Kristallmine?

a1exsh replied:
Sure thing.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Gems Mine"
-msgstr ""
+msgstr "Edelsteine Mine"
```
```suggestion
msgstr "Edelsteinmine"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Witch Doctor's Hut"
-msgstr ""
+msgstr "Hütte des Medizinmanns"
```
```suggestion
msgstr "Hütte des Hexendoktors"
```

dimag0g replied:
![Untitled](https://user-images.githubusercontent.com/10945319/148661154-f710f676-dbe2-4756-a6fe-b65cb7110e12.png)

a1exsh replied:
Well, I was remembering how it's called in the dialog when visiting the witch doctor: https://github.com/ihhub/fheroes2/pull/4905/files#diff-0210088403e77a015a387e50fda999eb4de2f2038e15178ca37a340843624153R4346

I firmly believe "Hexendoktor" is better than "Medizinmann" (esp. given it's not a man, but an orc). :D

a1exsh reviewed (commented):
75%

a1exsh commented on files/lang/de.po:
```diff
@@ -6821,13 +7839,23 @@ msgid ""
+"Feuerelement, das auf einem Felsen sitzt. Es schaut auf, sein brennendes "
+"Gesicht verzerrt sich in einem Blick starken Schmerzen. Es wirft dann ein "
+"leuchtendes Objekt bei Ihnen. Sie legen Ihre Hände auf, um es zu blockieren, "
+"aber es passiert direkt durch sie und separiert sich in die Brust."
```
```suggestion
"aber es passiert direkt durch sie und brannt sich in Ihre Brust an."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6836,20 +7864,35 @@ msgid ""
 "only man who was known to wear solid gold armor."
 msgstr ""
+"Sie sehen ein glänzendes Objekt, das aus dem Boden stößt. Sie senden ein "
+"Mitglied Ihrer Partei, um zu untersuchen. Er kommt mit einem goldenen Helm "
```
```suggestion
"Mitglied Ihrer Partei, um es zu untersuchen. Er kommt mit einem goldenen Helm "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6836,20 +7864,35 @@ msgid ""
+"Sie sehen ein glänzendes Objekt, das aus dem Boden stößt. Sie senden ein "
+"Mitglied Ihrer Partei, um zu untersuchen. Er kommt mit einem goldenen Helm "
+"in den Händen zurück. Sie erkennen, dass es der Helm des legendären Andurans "
+"sein muss, der einzige Mann, der dafür bekannt war, solide Goldpanzer zu "
```
```suggestion
"sein muss, des einzigen Mannes, der dafür bekannt war, solide Goldpanzer zu "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6836,20 +7864,35 @@ msgid ""
+"Zombies tödlich verwundet wurde. Er bittet Sie, seinen Hammer mitzunehmen "
+"und zu beenden, was er angefangen hat. Wenn Sie es abholen, beginnt es zu "
+"Summen, und dann wird alles verschwommen. Die Zombies liegen tot, der Hammer "
+"tropft mit Blut. Sie schnappen es an Ihren Gürtel."
```
```suggestion
"und zu beenden, was er angefangen hat. Wenn Sie ihn abholen, beginnt er zu "
"Summen, und dann wird alles verschwommen. Die Zombies liegen tot, der Hammer "
"tropft mit Blut. Sie schnappen ihn an Ihren Gürtel."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6858,31 +7901,51 @@ msgid ""
 msgstr ""
+"Wenn Sie einen kleinen Hügel keimen, kommen Sie auf einen lächerlichen Blick "
+"auf den Blick. Ein Sprite versucht, ein Zepter zu tragen, das fast so groß "
+"ist, wie es ist. Sie versuchen nicht zu lachen und fragen: \"brauche Hilfe?"
```
```suggestion
"Als Sie einen kleinen Hügel keimen, kommen Sie auf einen lächerlichen Blick. "
"Ein Sprite versucht es, ein Zepter zu tragen, das fast so groß "
"ist, wie er selbst. Sie versuchen nicht zu lachen und fragen: \"Hilfe gebraucht?"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6858,31 +7901,51 @@ msgid ""
 "hands you a tiny sphere. As soon as you grasp it, you feel the magical "
 "energy drain from your limbs..."
 msgstr ""
+"Sie halten auf, um einem Bauern zu helfen, fangen Sie eine runzelige Stute. "
```
```suggestion
"Sie halten auf, um einem Bauern zu helfen eine runzelige Stute zu fangen. "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6858,31 +7901,51 @@ msgid ""
 "inscription. It reads: \"Brains best brawn and magic beats might. Heed my "
 "words, and you'll win every fight.\""
 msgstr ""
+"Während Sie das Spiel erschrecken, finden Ihre Truppen einen geheimnisvollen "
```
Heir wird bestimmt von der Jagd gesprochen.  Game = "das Wild" in this sense.

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 msgstr ""
+"Ein Troll stoppt Sie und sagt: \"Bezahl mir 5.000 Gold, oder das Schwert von "
+"Anduran wird dich auf der Stelle töten.\" Sie verweigern. Der Troll packt "
+"das Schwert, das an seinem Gürtel hängt, schreien Schmerzen und läuft weg. "
```
```suggestion
"das Schwert, das an seinem Gürtel hängt, schreit voller Schmerz und läuft weg. "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 "you discover it to be the enchanted shovel of the Gravediggers, long thought "
 "lost by mortals."
 msgstr ""
+"In der Nähe wurde eine schmutzige Schaufel in einen Erdhügel gestoßen. Bei "
```
So it's either "Schaufel" or "Spaten", but let's be consistent, please? :)

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 
 msgid "Cannot move artifact"
-msgstr ""
+msgstr "Artefakt kann nicht bewegt werden"
```
```suggestion
msgstr "Artefakt kann nicht übergibt werden"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 
 msgid "Move %{name}"
-msgstr ""
+msgstr "%{name} bewegen"
```
```suggestion
msgstr "%{name} übergeben"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 
 msgid "Crystal"
-msgstr ""
+msgstr "Kristall"
```
```suggestion
msgstr "Kristalle"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 
 msgid "Fireblast"
-msgstr ""
+msgstr "Feuerstrahl"
```
```suggestion
msgstr "Feuerstoß"
```

dimag0g replied:
The spell name from GoG translation

a1exsh replied:
Hm, but it's obviously wrong.  The only meaning of "Strahl" is "beam", so it's definitely a poor choice here.  Let's not blindly copy names from one particular translation, esp. given that we know it's flawed (Kobolde, and so on).

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 
 msgid "Chain Lightning"
-msgstr ""
+msgstr "Blitzstrahl-Salve"
```
```suggestion
msgstr "Kettenblitzstrahl"
```
I find it more appropriate.  What is "Salve" anyway?

dimag0g replied:
The spell name from GoG translation. No idea what "Salve" is, but again: let's make a separate PR with names we want to change.

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
+"Verursacht einen Blitz elektrischer Energie, der eine ausgewählte Kreatur "
+"trifft, dann die nächste Kreatur mit halbem Schaden trifft, dann die nächste "
+"Kreatur mit wieder halbem Schaden trifft, und so weiter, bis er zu schwach "
+"wird, um Schaden anzurichten.  Warnung!  Dieser Zauber kann deine eigenen "
```
```suggestion
"wird, um Schaden anzurichten.  Warnung!  Dieser Zauber kann Ihre eigenen "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 msgid ""
 "Teleports the creature you select to any open position on the battlefield."
 msgstr ""
+"Teleportiert die Kreatur, die du auswählst, an eine beliebige freie Position "
```
beliebige => Ihrer Wahl (otherwise it reads like "random position")

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7954,855 @@ msgid ""
 "Increases the defense skill of the targeted creatures.  This is an improved "
 "version of Stoneskin."
 msgstr ""
+"Erhöht die Verteidigungsfähigkeit der anvisierten Kreaturen.  Dies ist eine "
```
```suggestion
"Erhöht die Verteidigungsfähigkeit der ausgewählten Kreaturen.  Dies ist eine "
```

a1exsh reviewed (commented):
90%

dimag0g commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "smiling upon you, it does nothing."
 msgstr ""
+"Sie haben ein uraltes und verwittertes Steingötzenbild gefunden.\n"
+"Sie soll den Besuchern Glück schenken, aber da die Sterne bereits auf Sie "
```
Shouldn't it be "ein ...bild" for an indefinite article in "an idol"?

dimag0g replied:
Shouldn't it be "ein ...bild" for an indefinite article in "an idol"?

a1exsh replied:
Yes, indefinite is appropriate.  What I'm pointing at is the beginning of the second sentence.  Since "Bild" is neutrum, we should start the sentence not with "Sie", but with "Es".

dimag0g commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "A drink at the well is supposed to restore your spell points, but you are "
 "already at maximum."
 msgstr ""
+"Ein Getränk am Brunnen soll Ihre Zauberpunkte wiederherstellen, aber Sie "
```
Google suggests "Ein Schluck", let's go with that.

dimag0g commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "oasis might help again if you fought a battle first."
 msgstr ""
+"Das Getränk an der Oase ist erfrischend, bietet aber keinen weiteren Nutzen. "
+"Die Oase könnte wieder helfen, wenn Sie zuerst eine Schlacht geschlagen "
```
Would Getränk be also wrong here? Like someone brought a can of Cola to drink it under the palm trees :)

dimag0g replied:
Would Getränk be also wrong here? Like someone brought a can of Cola to drink it under the palm trees :)

a1exsh replied:
Yes, I think "Getränk" is incorrect and should be replaced everywhere.

dimag0g commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 
 msgid "Artifact"
-msgstr ""
+msgstr "Artifakt"
```
Let's go with "Artefakt"

dimag0g replied:
Let's go with "Artefakt"

dimag0g replied:
Let's go with "Artefakt"

dimag0g replied:
Let's go with "Artefakt"

dimag0g commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "the gold or distribute the gold to the peasants for experience. Do you wish "
 "to keep the gold?"
 msgstr ""
+"Nachdem Sie das Gebiet durchkämmt haben, stoßen Sie auf ein verstecktes "
```
verborgenes

dimag0g commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 msgid "Roc"
-msgstr ""
+msgstr "Roc"
 
 msgid "Rocs"
-msgstr ""
+msgstr "Rocs"
```
Changing Golems to Goleme is one thing, but Rocvogel is a bit of a deviation from Roc/Rocs, which is used in GoG translation. I'd rather commit this as is in the first place, and then open a separate PR where we discuss which creatures could be renamed.

a1exsh replied:
OK

dimag0g commented:
I have force-pushed a commit with just the changes from the review.

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "A colorful Rogues' wagon stands empty here. Perhaps more Rogues will be here "
 "later."
 msgstr ""
+"Ein bunter Rogues-Wagen steht hier leer. Vielleicht werden später noch mehr "
```
That's just a suggestion. ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5338,2456 @@ msgid ""
 
 msgid "Wizard"
-msgstr ""
+msgstr "Zauberer"
```
These 3 closely similar classes are particularly tricky to translate.  Did you take these from GoG?  If so, let's keep it, otherwise I'd think a bit more on how to make Zauberer vs. Zauberin more distinct.  Recall that Wizard has some female heroes as well, which in German should be properly referred to as "Zauberin".

dimag0g replied:
Yep, that comes from GoG, and yes, we may want to improve this later.

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "Returns the caster to any town or castle currently owned."
 msgstr ""
+"Bringt den Zaubernden in jede Stadt oder Burg zurück, die er gerade besitzt."
```
```suggestion
"Bringt den Zaubernden in jene Stadt oder Burg zurück, die er gerade besitzt."
```
May sound better ("jede" = every).

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "Close Spellbook"
-msgstr ""
+msgstr "Zaubersprüche schließen"
```
```suggestion
msgstr "Zauberbuch schließen"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "game: remember last focus"
-msgstr ""
+msgstr "Spiel: den letzten Fokus erinnern"
```
```suggestion
msgstr "Spiel: den letzten Fokus merken"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "world: Crystal Ball also added Identify Hero and Visions spells"
 msgstr ""
+"Welt: Kristallkugel fügte auch die Zauber \"Held identifizieren\" und "
```
```suggestion
"Welt: Kristallkugel fügt auch die Zauber \"Held identifizieren\" und "
```
English should be updated as well: "added" => "adds".

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "world: Neutral armies scale with game difficulty"
-msgstr ""
+msgstr "Welt: Neutrale Armeen skalieren mit der Spielschwierigkeit"
```
```suggestion
msgstr "Welt: Neutrale Armeen steigen mit der Spielschwierigkeit auf"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "world: use unique artifacts for resource affecting"
-msgstr ""
+msgstr "Welt: einzigartiger Artefakte zur Ressourcenversorgung verwenden"
```
```suggestion
msgstr "Welt: einzigartige Artefakte zur Ressourcenversorgung verwenden"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "world: Wind/Water Mills and Magic Garden can be captured"
-msgstr ""
+msgstr "Welt: Wind-/Wassermühlen und Magischer Garten können erobert werden"
```
```suggestion
msgstr "Welt: Wind-/Wassermühlen und Magische Gärten können erobert werden"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "world: disable Barrow Mounds"
-msgstr ""
+msgstr "Welt: Vergrabener Karren deaktivieren"
```
```suggestion
msgstr "Welt: Vergrabenen Karren deaktivieren"
```
Akkusativ ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "heroes: allow transcribing scrolls (needs: Eye Eagle skill)"
-msgstr ""
+msgstr "Helden: Umschreiben von Schriftrollen erlauben (benötigt Augenadler)"
```
```suggestion
msgstr "Helden: Umschreiben von Schriftrollen erlauben (benötigt Adleraugen)"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "battle: show army order"
-msgstr ""
+msgstr "Schlacht: Armeeordnung anzeigen"
```
```suggestion
msgstr "Schlacht: Reihenfolge von Armeen anzeigen"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "game: offer to continue the game afer victory condition"
-msgstr ""
+msgstr "Spiel: weitere Spiel nach der Siegbedingung anbieten"
```
```suggestion
msgstr "Spiel: weiteres Spiel nach der Siegbedingung anbieten"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
 
 msgid "north-west"
-msgstr ""
+msgstr "Nord-west"
```
```suggestion
msgstr "Nordwest"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7955,855 @@ msgid ""
+msgstr "Eine unbekannte Macht wird wiederbelebt..."
 
 msgid ""
 "Check the newest version of game at\n"
```
```suggestion
"Check the newest version of the game at\n"
```

a1exsh reviewed (commented):
100%!

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148675655-ff4d16aa-ba4f-436a-a10e-c9b2cd4803cf.png)

As expected with German, some words are quite long...  "Einstellungen" need to be made shorter (or the dialog window — wider).

a1exsh commented:
Roland's campaign, first scenario:
![image](https://user-images.githubusercontent.com/489601/148675938-03726229-a04c-401c-8d37-9a517f46275c.png)

1. The description doesn't fit (though maybe it would just fit, provided the umlauts were supported).
2. The scenario bonus "Kleine Schriftrolle" is truncated, but one more letter could actually fit easily: maybe we should update the code to allow that.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676063-e86087f2-eb9e-4eb5-a9ef-508c5f213856.png)

Bonus name truncated: "Erdwesen beschw".

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676101-f71ea4e5-a5f2-41cc-a76d-dc73b13440d4.png)

1. Description doesn't fit.
2. Award name "Gilde der Zauberinnen" is truncated.  Name it "Zauberinnengilde"?

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676146-6c10f470-b6ac-4fa3-9fb6-a224717b1e08.png)

Description doesn't fit.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676163-98b43c3f-ef97-4f62-8984-95d7757a2aed.png)

1. Award "Lord Corlagon besiegt" is truncated.  Make it just "Corlagon besiegt"?
2. Bonus "Scheussliche Maske" is truncated.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676225-27ed80a3-4ce0-4361-b2ad-a55e2d62fb9f.png)

Description doesn't fit.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676251-5926ac73-3b45-4f74-8e89-68c8cc988db8.png)

Description doesn't fit.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676294-44a486a7-1247-4128-87b7-74cfdcf7bcac.png)

1. Description doesn't fit.
2. "Verteidigungshelm" doesn't fit, but we could fix the code to allow it, probably.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676340-49804077-ca98-4da5-95e6-4fc35e783133.png)

Award name doesn't fit :(

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676385-09b200cf-79bf-41aa-b328-427d8ed55983.png)

Description doesn't fit.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148676455-5cff7c6d-8db9-4fc0-a83a-629b3a862080.png)

Bonus name: "Medaille der Tapferkeit".

a1exsh commented:
1. Hero stats are really messy due to long words:
   ![image](https://user-images.githubusercontent.com/489601/148676561-6843d593-aa41-432e-8d10-f9b4f63df791.png)

2. "Hoch Moral" is incorrect, should be "Hohe":
   ![image](https://user-images.githubusercontent.com/489601/148676648-fef9a3be-a782-41e3-a7d0-550a332aa64f.png)
   The same for luck (should be "Normales Glück"):
   ![image](https://user-images.githubusercontent.com/489601/148676702-c882f1e4-88c5-44dc-890b-7bdc6259c42f.png)

3. Zauberspruchpunkte is too long for this window:
   ![image](https://user-images.githubusercontent.com/489601/148676824-d0f613e8-7265-43d3-9476-198812da1b4f.png)

4. "Verbleibende Trefferpunkte":
   ![image](https://user-images.githubusercontent.com/489601/148676871-24cc3bcf-fe59-4b4d-9c59-a24707f27c38.png)

5. "Zug 1".  "Runde" is a better fit here:
   ![image](https://user-images.githubusercontent.com/489601/148676905-1a868557-3ef1-498d-9010-83ed193a2cae.png)

6. Better: "Sie dürfen lernen entweder: ... oder ...".
   ![image](https://user-images.githubusercontent.com/489601/148677003-4075d01f-c160-4e60-ac76-6673ae5431e5.png)

7. Better: "Halblinge rekrutieren".
   ![image](https://user-images.githubusercontent.com/489601/148677038-ad4305d8-f5be-47cd-95cb-af482060b21f.png)
   "Anzahl zu kaufen" is a bit too wide.  "Anzahl"?  Esp. since we are not buying them, rather recruit, as the dialog title says.

8. "Mandigal _die_ Zauberer...".  Why "die"? :D
   ![image](https://user-images.githubusercontent.com/489601/148677119-e5d42b12-857a-4549-8cdf-e0370e8dc3fe.png)

9. "Verteidigung" doesn't fit:
   ![image](https://user-images.githubusercontent.com/489601/148677197-c2ee8788-d90b-4097-aa2b-5b940148f82c.png)

10. "Verteidiger: Nichts".  Better: "Niemand" (no one).
  ![image](https://user-images.githubusercontent.com/489601/148677243-613ceb75-e3b3-47bb-9d75-ff6c46eaeb8d.png)

a1exsh commented:
1. "Der Gießerei" is incorrect.  Should be "Die".  The same problem is with: Wohnhaus (das), Felsennest (das), Wolkenburg (die).
   ![image](https://user-images.githubusercontent.com/489601/148677279-fc23744c-e6bc-4631-b153-5fac4b603e83.png)
   Also, the conversion of the monster name to lowercase is wrong for German, should be: "... produziert Eisengoleme".

2. Ah, turns out this message is used not for meeting the heroes, but rather in View hero.  "Übergeben" is incorrect here.  Let's use "umstellen" or something similar.
   ![image](https://user-images.githubusercontent.com/489601/148677428-5cafb7f6-395f-43a2-9065-06719bf0485a.png)

3. The dialog title is too wide:
![image](https://user-images.githubusercontent.com/489601/148677503-dc81efbf-1851-40e6-9cd9-9712faedec98.png)

4. The status text becomes too wide here:
   ![image](https://user-images.githubusercontent.com/489601/148677809-6627fd02-a56b-41b9-9133-f269492f4ada.png)

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148681938-7ba1a6d5-a901-4d06-80be-f41a2e8b7a6c.png)

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148682421-551ff037-4b6f-420d-b4a6-b6e91fd2a7e1.png)

1. Scenario title is a bit too wide to fit: "Die ewigen Schriftrollen".
2. Bonus name: "Herausragenste..."

ihhub commented:
> ![image](https://user-images.githubusercontent.com/489601/148675655-ff4d16aa-ba4f-436a-a10e-c9b2cd4803cf.png)
> 
> As expected with German, some words are quite long... "Einstellungen" need to be made shorter (or the dialog window — wider).

We can make the window wider 🙂

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148683786-024505cf-1d4e-46ab-a762-c754ec59cb0b.png)

Mal wieder ein Nomen kleingeschrieben... nicht gut für Deutsch. ;)

All monster names seem to be lower-cased in a number of places in the game.

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148688273-122c0e21-f5a1-4403-b818-c6a2ed6c56f1.png)

Description does't fit.

a1exsh commented:
Artifact placeholder didn't work, because it got accidentally translated to German:
![image](https://user-images.githubusercontent.com/489601/148690294-5155e958-d537-4626-9a06-14d97e75cc23.png)

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148690351-a84a66eb-6094-4543-a4c6-a6ffb045bc29.png)

Award truncated: "Zauberkugel der Auf..."

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/148693356-6003e647-ba0c-478b-ab7b-6a8fc5446e7b.png)

a1exsh commented:
@dimag0g In general it is pretty much playable, except for a few places where the text is too wide to fit, which I've mentioned.  Great job! :)

dimag0g commented:
@a1exsh Thanks! I'll need time to make fixes (I was in Elternzeit which is now over, so you can imagine I have some rather busy evenings :). Just a couple more days until I find enough time and motivation.

a1exsh commented:
@dimag0g no rush :) take your time!

dimag0g commented:
@a1exsh I tried to make some of the translations shorter, could you take a look at these?

a1exsh commented:
@dimag0g I hope to find the time later this week — thanks :)

dimag0g commented:
@a1exsh Before we can close this, here are a few remaing points I'd like to clarify with you

1. Please take a look at the last changeset. It has new scenario description translations and a few other things I'd like you to review.
2. I translated "%{name} the %{race}" as "%{name} (%{race})" to make it more gender-neutral, do you think it's OK?
3. I translated "skip" as in "Skip this turn" as "überspringen". Is it OK or do you prefer "aussetzen"?
4. "Das Blut ist dicker" is too long for the interface, I reverted it back to "Blut ist dicker", OK?
5. I replaced "Die Drachen, knurrend und knurrend" with "Die Drachen, knurrend und grollend...", OK?
6. Is it OK to say "%{offer} von %{total} %{monster}" and "Alle %{offer} der %{monster}" as in "5 of 10 goblins will join your army"?
7.  I removed the word "Spieler" from the sentences mentioning color. There is no way of getting a grammatically correct sentence for every color otherwise. OK?
8. I translated "hidden" as "verborgenes" when talking about "Versteck" / "Schatzversteck", so as not to repeat the same root twice. OK?
9. I translated "a small pot filled with precious things" as "[zu[ einem kleinen Topf mit wertvollen Dingen drin". OK?
10. I translated "a drink" as "Der Schluck" when talking about wells/oasis/watering holes. OK?
11. Could you please clarify this comment: "Während Sie das Spiel erschrecken, finden Ihre Truppen einen geheimnisvollen "
>Heir wird bestimmt von der Jagd gesprochen. Game = "das Wild" in this sense.

Regarding missing capitals: some of the strings (mostly monster names) are reused by being converted to lower case in the game, so that you see "Elves" in the upgrade dialog but "The elves did 125 damage" during a battle. This is wrong in German, but it cannot be fixed in translation files.

Monster / spell names stay as in GoG, except for:
- "Kobolde" -> "Feen" 
- "Medizinmann"->  "Hexendoktor" 
- "Feuerstrahl" -> "Feuerstoß"
- "Blitzstrahl-Salve" -> "Kettenblitzstrahl".
- Some heroes names were germanized (Natascha, Kräger)

a1exsh commented:
> @a1exsh I updated the last commit again. Now I won't touch it until you review. Before we can close this, here are a few remaing points I'd like to clarify with you
> 
>     1. Please take a look at the last commit. It has new (shorter) scenario description translations and a few other things I'd like you to review.
> 
>     2. I translated "%{name} the %{race}" as "%{name} (%{race})" to make it more gender-neutral, do you think it's OK?

One occurrence left here: https://github.com/ihhub/fheroes2/blob/214d5c4ce32aad3dd1110d8b8fd0995f1a0c935b/files/lang/de.po#L652

>     3. I translated "skip" as in "Skip this turn" as "überspringen". Is it OK or do you prefer "aussetzen"?

Actually, "aussetzen" is preferable, at least from what I know of the use of words in board games.

>     4. "Das Blut ist dicker" is too long for the interface, I reverted it back to "Blut ist dicker", OK?

Fine.

>     5. I replaced "Die Drachen, knurrend und knurrend" with "Die Drachen, knurrend und grollend...", OK?

:+1: 

>     6. Is it OK to say "%{offer} von %{total} %{monster}" and "Alle %{offer} der %{monster}" as in "5 of 10 goblins will join your army"?

OK

>     7. I removed the word "Spieler" from the sentences mentioning color. There is no way of getting a grammatically correct sentence for every color otherwise. OK?

That is actually an easy one as in German you just swap the order of words: "Blue player" becomes "Spieler Blau", which is perfectly correct in this context. ;)

>     8. I translated "hidden" as "verborgenes" when talking about "Versteck" / "Schatzversteck", so as not to repeat the same root twice. OK?

Awesome!

>     9. I translated "a small pot filled with precious things" as "[zu[ einem kleinen Topf mit wertvollen Dingen drin". OK?

OK

>     10. I translated "a drink" as "Der Schluck" when talking about wells/oasis/watering holes. OK?

:+1: 

>     11. Could you please clarify this comment: "Während Sie das Spiel erschrecken, finden Ihre Truppen einen geheimnisvollen "
> 
> > Heir wird bestimmt von der Jagd gesprochen. Game = "das Wild" in this sense.

The word "game" has more than one meaning in English.  In this context it means animals being hunted upon ("das Wild" in German, or, if you know Russian — "дичь").

So it could be something like: "Als Ihre Truppen das Wild verjagten, ...".

> Regarding missing capitals: some of the strings (mostly monster names) are reused by being converted to lower case in the game, so that you see "Elves" in the upgrade dialog but "The elves did 125 damage" during a battle. This is wrong in German, but it cannot be fixed in translation files.

Yep, let's fix it separately by updating the relevant code: nothing we can do in translation for that right now.

> Monster / spell names stay as in GoG, except for:
> 
>     * "Kobolde" -> "Feen"
> 
>     * "Medizinmann"->  "Hexendoktor"
> 
>     * "Feuerstrahl" -> "Feuerstoß"
> 
>     * "Blitzstrahl-Salve" -> "Kettenblitzstrahl".
> 
>     * Some heroes names were germanized (Natascha, Kräger)

Awesome! :)

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 
 msgid "Necroman"
-msgstr ""
+msgstr "Totenbesc"
```
```suggestion
msgstr "T.beschw."
```
Not ideal, but correctly abbreviated, IMO.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "Gargoyles"
-msgstr ""
+msgstr "Wasserspeiern"
```
```suggestion
msgstr "Wasserspeier"
```
Not sure if I suggested it, but this form is correct plural.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5337,2456 @@ msgid ""
 
 msgid "morale|Treason"
-msgstr ""
+msgstr "Verrate"
```
```suggestion
msgstr "Verratende"
```
At least try to make it look like an adjective. ;)

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 msgstr ""
+"Die %{monster}, erschrocken über die Macht Ihrer Kräfte, beginnen sich zu "
+"zerstreuen.\n"
+"Willst du sie verfolgen und angreifen?"
```
@dimag0g Please double-check once again all uses of polite/normal "you".

a1exsh commented on files/lang/de.po:
```diff
@@ -3231,864 +3673,1121 @@ msgid ""
 "quantity of %{resource}."
 msgstr ""
+"Sie finden eine kleine\n"
+"Menge von %{resource}."
```
```suggestion
"Menge an %{resource}."
```
Sounds a little bit nicer.

a1exsh commented:
"Zauberkraft" _almost_ fits, but "Bewegungspunkte" is still too wide:
![image](https://user-images.githubusercontent.com/489601/149558110-5ff76d5a-2c0a-448b-b2d4-df51659e0ae2.png)

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/149561629-fad81569-c793-4462-a74e-979f2686a1da.png)

Correct: "... Monat des Vampirs."

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/149563247-fc5fbf99-2e90-41c0-8e3f-7e31478304f5.png)

"Die Schwarze Perle ..."

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/149563354-642524bf-8442-4ec1-a799-d8643858e3c6.png)

"Der (G)goldene Bogen ..."

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/149563498-74263123-3e1b-4ef7-b8cb-cbeac48ed631.png)

"... von einem (G)grünen Drachen ..."

a1exsh commented:
Multiple text elements don't fit :(
![image](https://user-images.githubusercontent.com/489601/149565688-5f22b954-ba74-45a5-88cf-489943b771c9.png)

a1exsh commented:
![image](https://user-images.githubusercontent.com/489601/149571581-cb902b2a-ad18-4a80-bded-3e9979c905de.png) ![image](https://user-images.githubusercontent.com/489601/149571999-d69326fe-15c9-4f91-8b61-8c0519ca7830.png) ![image](https://user-images.githubusercontent.com/489601/149572155-7b8851ae-7c4b-4ab3-b277-a18b21d60743.png) ![image](https://user-images.githubusercontent.com/489601/149572271-36fcbfbe-5928-4ea9-8cfd-e48337a2ae6b.png) ![image](https://user-images.githubusercontent.com/489601/149573219-8d6c920f-bdfb-4819-a5b6-b578715fb125.png)

![image](https://user-images.githubusercontent.com/489601/149573419-9b11d4b0-f4af-4856-a777-39fdec544ffa.png) ![image](https://user-images.githubusercontent.com/489601/149573912-78a9755d-d122-4cea-a3a1-b6e513916f66.png) ![image](https://user-images.githubusercontent.com/489601/149574036-d7df61bc-69db-4cae-b765-98865919c26f.png) ![image](https://user-images.githubusercontent.com/489601/149574347-58e58bef-9f7d-4efb-8111-b146c3f542d7.png)

dimag0g commented:
@a1exsh  I made more fixes. Among others, I replaced "schwierigkeit" with "grad" to make it fit, and shortened long stats to "Z-Kraft" und "BW-Punkte". I didn't fix the spell names, as these are used in the spellbook (where they need to be multi-lined correctly), but also in status messages and such, where multi-line would ruin formatting.

a1exsh commented on files/lang/Makefile:
```diff
@@ -44,9 +44,10 @@ de.mo: de.po
 	sed -e 's/ä/ae/g' $< > $<.tmp
 	sed -i~ -e 's/ö/oe/g' $<.tmp
 	sed -i~ -e 's/ü/ue/g' $<.tmp
+	sed -i~ -e 's/Ä/Ae/g' $<.tmp
```
```suggestion
	sed -i~ -e 's/Ä/Ae/g' $<.tmp
	sed -i~ -e 's/ß/ss/g' $<.tmp
```

dimag0g replied:
Replacing `ß` to `ss` is not necessary, as it is correctly done by `iconv`. The other lines are there because `iconv` replaces  `ä` to `a:` on many platforms, and this (1) looks ugly in German, and (2) ruins the formatting of .po files.

a1exsh replied:
Ah, OK then.

a1exsh commented:
I also see the same problems with the spell names in the spellbook:
![image](https://user-images.githubusercontent.com/489601/149722525-b9e19eb8-32ea-4259-9e61-c07f1794636f.png)
![image](https://user-images.githubusercontent.com/489601/149722592-bec047fc-41f6-4445-ac2a-f048db5963e5.png)

a1exsh commented:
> "Z-Kraft" und "BW-Punkte"

These two look really odd. :-/  At the same time I notice, that we've got plenty of space in the window showing the hero stats on the adventure map, so maybe we should adjust the code to fit a bit longer labels here?

a1exsh commented:
It looks like it's not enough to fix it in one place, because the other use becomes incorrect (here in the title it should still be "Goldener Bogen" or "Der Goldene Bogen"):
![image](https://user-images.githubusercontent.com/489601/149723289-1de94151-434d-4218-aaa4-e26f2d0aab7e.png)

Should be: "_Die_ Glücksmünze ...":
![image](https://user-images.githubusercontent.com/489601/149723444-dbe2717c-c1aa-45a2-93f2-d6b5866e5ff2.png)

_Das_ Buch:
![image](https://user-images.githubusercontent.com/489601/149723624-ebd2c003-141b-4770-8290-f1c1cf9e0c50.png)

_Die_ Rolle:
![image](https://user-images.githubusercontent.com/489601/149723740-d96a2f01-1b18-46e8-927b-8b11c51ac325.png)

_Die_ Pfote:
![image](https://user-images.githubusercontent.com/489601/149723865-fa3e7ff4-1f5c-44bf-baa7-81229e8c3eed.png)

And so on...

dimag0g commented:
The articles are unfixable in the current situation: the same strings "The %{name} increases xxx by %{count}" are used with e.g. "Die Perle" and "Der Stab", so whatever article we use it will be wrong in some cases.

a1exsh commented on files/lang/de.po:
```diff
@@ -4240,134 +5005,197 @@ msgid ""
 "to you for %{count} gold.\""
 msgstr ""
+"Der Dämon springt dich an und hat seine Klauen an Ihrer Kehle, bevor du "
+"überhaupt dein Schwert ziehen kannst. \"Dein Leben gehört mir\", sagt er. "
```
```suggestion
"Der Dämon springt Sie an und hat seine Klauen an Ihrer Kehle, bevor Sie "
"überhaupt Ihren Schwert ziehen können. \"Dein Leben gehört mir\", sagt er. "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5339,2456 @@ msgid ""
 "so here's your reward. Now begone."
 msgstr ""
+"Die Sphinx sieht etwas enttäuscht aus und seufzt. Du hast mein Rätsel "
+"gelöst, hier ist deine Belohnung. Und nun verschwinde."
```
```suggestion
"Die Sphinx sieht etwas enttäuscht aus und seufzt. \"Du hast mein Rätsel "
"gelöst, hier ist deine Belohnung. Und nun verschwinde.\""
```
Quotes are also missing in the original text.

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5339,2456 @@ msgid ""
+"Eine magische Barriere steht hoch vor Ihnen und versperrt Ihnen den Weg. Auf "
+"dem Bogen stehen Runen,\n"
+"\"Sprich den Schlüssel und du kannst passieren.\"\n"
+"Du sprichst, und nichts passiert."
```
```suggestion
"Sie sprechen, aber es passiert nichts."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4461,2331 +5339,2456 @@ msgid ""
+"Eine magische Barriere steht hoch vor Ihnen und versperrt Ihnen den Weg. Auf "
+"dem Bogen stehen Runen,\n"
+"\"Sprich den Schlüssel und du darfst passieren.\"\n"
+"Wenn Sie das magische Wort sprechen, löst sich die glühende Barriere in "
```
```suggestion
"Als Sie das magische Wort sprechen, löst sich die glühende Barriere in "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7956,855 @@ msgid ""
 msgid ""
 "Teleports the creature you select to any open position on the battlefield."
 msgstr ""
+"Teleportiert die Kreatur, die du auswählst, an eine freie Position Ihrer Wahl "
```
```suggestion
"Teleportiert die Kreatur, die Sie auswählen, an eine freie Position Ihrer Wahl "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7956,855 @@ msgid ""
 msgstr ""
+"Ruft das nächstgelegene unbesetzte, befreundete Boot an einen benachbarten "
+"Ort am Ufer.  Ein befreundetes Boot ist eines, das du gerade gebaut hast "
+"oder das du als letzter Spieler besetzt hast."
```
```suggestion
"Ort am Ufer.  Ein befreundetes Boot ist eines, das Sie gerade gebaut haben "
"oder das Sie als letzter Spieler besetzt haben."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4098,140 +4799,204 @@ msgid ""
 msgstr ""
+"Ein paar Trolle bleiben zurück und kauern unter der Brücke. Sie kommen auf "
+"dich zu und bieten dir an, sich deinen Truppen als Söldner anzuschließen. "
+"Wollt Ihr Trolle kaufen?"
```
```suggestion
"Sie zu und bieten Ihnen an, sich Ihren Truppen als Söldner anzuschließen. "
"Wollen Sie Trolle kaufen?"
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3135 @@
 "will be yours."
 msgstr ""
+"Beseitigen Sie alle Gegner in diesem Gebiet. Dann gehört das erste Stück des "
+"Artefakts dir."
```
```suggestion
"Artefakts Ihnen."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3135 @@
 msgid "The ogres recognize you as the Dwarfbane and lumber over to join you."
 msgstr ""
+"Die Oger erkennen Sie als die Zwergenfluch und humpeln zu dir hinüber, um "
+"sich dir anzuschließen."
```
```suggestion
"Die Oger erkennen Sie als die Zwergenfluch und humpeln zu Ihnen hinüber, um "
"sich Ihnen anzuschließen."
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3135 @@
 
 msgid "The dwarves recognize their allies and gladly join your forces."
 msgstr ""
+"Die Zwerge erkennen ihre Verbündeten und schließen sich gerne ihren Kräften "
```
```suggestion
"Die Zwerge erkennen ihren Verbündeten und schließen sich gerne Ihren Kräften "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -1,2786 +1,3135 @@
 "says, \"Let us not impede your progress, ally!  Move on, and may victory be "
 "yours.\""
 msgstr ""
+"Die Elfen stehen stramm, als Sie sich ihnen näheren.  Ihr Anführer ruft dir "
```
```suggestion
"Die Elfen stehen stramm, als Sie sich ihnen näheren.  Ihr Anführer ruft Ihnen "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6836,20 +7866,35 @@ msgid ""
 msgstr ""
+"Plötzlich durchdringt eine beißende Kälte Ihren Körper. Sie ergreifen auf "
+"und fallen von Ihrem Pferd. Der Schmerz senkt, aber Sie fühlen immer noch "
+"so, als ob Ihre Brust eingefroren ist. Wenn Sie sich vom Boden abholen, "
```
```suggestion
"so, als ob Ihre Brust eingefroren ist. Als Sie sich vom Boden abholen, "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6836,20 +7866,35 @@ msgid ""
 msgstr ""
+"Sie kommen auf eine Schlacht, in der ein Paladin von einer Gruppe von "
+"Zombies tödlich verwundet wurde. Er bittet Sie, seinen Hammer mitzunehmen "
+"und zu beenden, was er angefangen hat. Wenn Sie ihn  abholen, beginnt er zu "
```
```suggestion
"und zu beenden, was er angefangen hat. Als Sie ihn abholen, beginnt er zu "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -6891,671 +7956,855 @@ msgid ""
+"Ein Troll stoppt Sie und sagt: \"Bezahl mir 5.000 Gold, oder das Schwert von "
+"Anduran wird dich auf der Stelle töten.\" Sie verweigern. Der Troll packt "
+"das Schwert, das an seinem Gürtel hängt, schreit voller Schmerz und läuft weg. "
+"Wenn Sie das fähige Schwert abholen, danken Sie, dass halbgeteitete Trolls "
```
```suggestion
"Als Sie das fähige Schwert abholen, danken Sie, dass halbgeteitete Trolls "
```

a1exsh commented on files/lang/de.po:
```diff
@@ -4098,140 +4799,204 @@ msgid ""
 "whisper: \"Why have you come? Are you here to call upon the forces of the "
 "air?\""
 msgstr ""
+"Weiße Steinsäulen stützen ein Dach, das sich in den Himmel erhebt. Wenn Sie "
```
```suggestion
"Weiße Steinsäulen stützen ein Dach, das sich in den Himmel erhebt. Als Sie "
```

a1exsh reviewed (commented):
@dimag0g I've corrected all use of normal "you" to the polite one, which were still left.  Also found a few more places where "wenn" should be changed to "als".

a1exsh commented:
> The articles are unfixable in the current situation: the same strings "The %{name} increases xxx by %{count}" are used with e.g. "Die Perle" and "Der Stab", so whatever article we use it will be wrong in some cases.

Yeah, I expected that much.  Let's fix it later together with adapting the original string (by making "The" part of the English name, for example).

a1exsh commented:
@dimag0g I think we are pretty close now.  Let's see what can be done about the hero stats and then we can ask @ihhub to merge it. :)

dimag0g commented:
I've implemented more of your fixes. Let's keep the quick info labels short and ugly for now. I see there is a way to move the stat numbers to the right by increasing the constant in `dst_pt.x += 75` lines in `dialog_quickinfo.cpp`. Let's do this as a separate PR and expand abbreviated strings in translations.

a1exsh commented:
@dimag0g agree: let's call it good for now. :)  My browser crashes when working with such a huge PR already...

dimag0g commented:
@ihhub Could you please merge this?

ihhub commented:
@dimag0g , thank you so much for great efforts for German translation!
